tests/test_bulk_tx_basic_noack/src/main.xc

```diff
@@ -1,114 +1,13 @@
 // Copyright 2016-2021 XMOS LIMITED.
 // This Software is subject to the terms of the XMOS Public Licence: Version 1.
-/*
- * Test the use of the ExampleTestbench. Test that the value 0 and 1 can be sent
- * in both directions between the ports.
- *
- * NOTE: The src/testbenches/ExampleTestbench must have been compiled for this to run without error.
- *
- */
-#include <xs1.h>
-#include <print.h>
-#include <stdio.h>
-#include "xud.h"
-#include "platform.h"
-#include "xc_ptr.h"
+#include "shared.h"
 
-#define XUD_EP_COUNT_OUT   3
-#define XUD_EP_COUNT_IN    3
+#ifndef PKT_LENGTH_START
+#define PKT_LENGTH_START 	(10)
+#endif
 
+#ifndef PKT_LENGTH_END
+#define PKT_LENGTH_END 		(14)
+#endif
 
```

Comment:
Missing definition of `test_func()`?

Reply:
I don't understand this comment  - the #include includes all code from another test.